data_structures/stack/stack.c

```diff
@@ -42,7 +42,7 @@ void initStack()
     grow: increases the stack by 10 elements.
           This utility function isn't part of the public interface
 */
-void grow()
+void  grow(void ** oldMemory)
 {
```

Comment:
I think you do not need `void ** oldMemory`
just free tmp

Reply:
Yes， I will  change it.